helm/defectdojo/values.yaml

```diff
@@ -266,6 +266,7 @@ mysql:
   secretKey: mysql-password
   mysqlDatabase: defectdojo
   service:
+    port: 3306
```

Comment:
From next release, this port is not in use anymore. @alles-klar can give you more information

Reply:
Shall it be removed? Because it is referenced it all deploymets at the moment as .Values.mysql.service.port

Reply:
after the rebase you will find the value (mysqlmaster.service.port), see https://github.com/DefectDojo/django-DefectDojo/pull/5450

Reply:
if the rebase is messy you can try to create a new PR  from a new branch if that's easier.

Reply:
I'll close this one, because I created a new one: https://github.com/DefectDojo/django-DefectDojo/pull/5534